core/frontend/src/components/common/ErrorDialog.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-dialog 
+    v-model="show_dialog"
+    width="500"
+    transition="dialog-bottom-transition"
+  >
+    <v-card>
+      <v-card-title> Error </v-card-title>
+      <v-card-text> {{ error_message }} </v-card-text>
+      <v-card-actions>
+        <v-spacer />
+        <v-btn
+          text
+          @click="ignoreError"
+        >
+          Ignore this error
+        </v-btn>
+        <v-btn
+          text
+          @click="closeError"
+        >
+          Close
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { getModule } from 'vuex-module-decorators'
+import CommonStore from '@/store/common'
+import { Error } from '@/types/common'
+
+const common_store: CommonStore = getModule(CommonStore)
+
+export default Vue.extend({
+  name: 'Error',
+  computed: {
+    error (): Error | null {
+      return common_store.current_error
+    },
+    error_message (): string {
+      return this.error === null ? '' : this.error.message
+    },
+    show_dialog: {
+      get (): boolean {
+        return this.error === null ? false : true
+      },
+      set (value: boolean): void {
```

Comment:
`true` is not supported ? If so this should not receive arguments.

Reply:
Didn't get it. get/set for are an existing interface that I'm implementing.